tests/db/pg_client.py

```diff
@@ -1,16 +1,47 @@
-import os
-
 from dotenv import load_dotenv
 import psycopg2
+from psycopg2._psycopg import cursor, connection
+from duneapi.types import DuneQuery
+from psycopg2.extras import RealDictCursor
 
 
 def connect():
     load_dotenv()
     conn = psycopg2.connect(
-        host=os.environ["POSTGRES_HOST"],
-        port=os.environ["POSTGRES_PORT"],
+        host="localhost",
+        port=5432,
         database="postgres",
         user="postgres",
         password="postgres",
     )
     return conn
+
+
+def populate_db() -> tuple[connection, cursor]:
+    conn = connect()
+    cur = conn.cursor(cursor_factory=RealDictCursor)
```

Comment:
`conn` and `cur` are industry standards for these variables. Also, we can't use cursor here because it conflicts with the types (see the function signature).

Reply:
I think I might be able to fix the type conflict by importing their types library. Let me see.

Reply:
Unfortunately, the types we have here are indeed the types (with the unusual lower case). I don't think we can get around this at the moment.

Reply:
Sorry.